crates/sui-framework/packages/sui-framework/sources/address.move

```diff
@@ -1,12 +1,15 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+#[defines_primitive(address)]
 module sui::address {
```

Comment:
Can we make `std::address` the definer instead? this feels a bit strange.

Or do you want to make that a separate PR?

Reply:
I would love to, but I talked with Tim some and it seems like migrating the native functions is going to be a large and arduous ordeal (plus we'll have to retain them in some way in both places to avoid breaking existing code). I thought it would be best to do it as a follow-up diff (though it's possible marking this as the primitive defining and providing the use fun here will lead to more dragons).